### ISSUE TITLE
CSS and JavaScript should be minified where possible

### ISSUE THREAD

**Issue by osg74** [akvo/akvo-web#175]
As per site performance testing: http://developers.google.com/speed/pagespeed/insights/?url=http%3A%2F%2Fakvo.org%2F&tab=desktop
- https://developers.google.com/speed/docs/insights/OptimizeCSSDelivery
- https://developers.google.com/speed/docs/insights/MinifyResources

**Comment by iperdomo** [comment]
@Lynngre i see a `Nice to have` tag on this issue. Performance should be tagged a `Must have`.

e.g. http://www.fastcompany.com/1825005/how-one-second-could-cost-amazon-16-billion-sales

**Comment by Lynngre** [comment]
@iperdomo yes totally agree this is important, unfortunately due to lack of resources for the website & priority being focused on getting the content out this has fallen on the back burner.  Not sure if @osgl and @carlio have any ideas on this will followup with them.

**Comment by iperdomo** [comment]
There are few javascript resources that can be used in the minified version already, e.g.

Instead of:
https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.11.2/jquery-ui.js

Use:
https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.11.2/jquery-ui.min.js

In fact this JS file is included more than once.

**Comment by Lynngre** [comment]
@osgl @orifito @carlio do you have any input on this issue?

**Comment by gabemart** [comment]
@osgl Could someone from devops look into getting the w3 total cache plugin activated properly on akvo-dev? 

By "activated properly", I just mean running with the appropriate permissions to function, not necessarily serving pages correctly. Currently, trying to activate the plugin results in `FTP credentials don't allow to write to file` and `W3 Total Cache Error: Files and directories could not be automatically created to complete the installation.`

If we could get the plugin activated correctly, I could then experiment to see if there's a combination of settings that does some concatenation / minification without breaking anything, which would be an easy win.

**Comment by Lynngre** [comment]
@gabemart just a note that W3 Cache was disabled as it was rendering an empty homepage to visitors after the initial visit.

**Comment by Lynngre** [comment]
@osgl could we possibly chat about this 'minify' issue when I'm in London next week?

**Comment by Lynngre** [comment]
https://github.com/akvo/akvo-web/issues/595

**Comment by Lynngre** [comment]
W3 total cache will be updated in the v1.1.14 Omnilingual Onion release but the plugin cannot be activated as yet as there are rewrite issues @orifito is looking into for us.

**Comment by Lynngre** [comment]
#595
